### DISCUSSION TITLE
Danfoss Allly thermostat using external room temperature

### DISCUSSION THREAD

**Discussion by jeppper** [Koenkk/zigbee2mqtt#14794] [Devices]
Anyone knows if using a external room tempearure is possible instead of the local measured by the thermostat?

**Comment by lux73** [comment]
yes it is

**Comment by jeppper** [comment]
But how? 
When setting "radiator covered" to true and changing the "external measured room sensor" with the slider under "exposes" nothing happens.

**Reply by lux73** [reply]
anything is described here -> https://github.com/Koenkk/zigbee2mqtt/issues/4504#issuecomment-985480553

in short: your preferred Home Automation must send the external measured Temperature in regulary Timeslots to your Danfoss

does not work good for me, my external Aqara Temperature Sensors are not sensitive enough at all - so my Room will overheat with external Temperature Sensor :-(

**Comment by jeppper** [comment]
Okay. I have the same setup with Aqara sensors :-( So it can´t be used in practise.
You just use the internal measurement instead then?

Also it doesn´t seem possible to change modes on the thermostat. I would like to have a comfort and standby mode for changing the setpoint whether the room is occupied or not. I guess it must be done manually by changing the setpoint instead :-/

**Reply by lux73** [reply]
oh please, the fact that's not working properly for me does not mean it works not for you ;-)

yes i'm using internal measurement - but i had to work out that my TRV are too close to my Radiator. With this and the fact that's the Ally's are really sensitive with temperature measuring i had to manage more Distance between my Radioator and the Ally's

so i have mounted some Distance Adapter -> https://www.amazon.de/Heimeier-Spindel-Verl%C3%A4ngerung-Thermostat-Unterteile/dp/B00IHEZ3VC?content-id=amzn1.sym.995b3ade-87fd-4685-9950-c7995e7b483c&psc=1

and for another Radiator some Bracket Adapter -> https://www.amazon.de/Oventrop-1011450-Winkeladapter-Ventil-HK-wei%C3%9F/dp/B0017V3P4O?content-id=amzn1.sym.995b3ade-87fd-4685-9950-c7995e7b483c&psc=1

with this Adapters my Ally have more Space between the Radiator and working really really sensitive & reliable :-)

without these there are to much Heat measured from the Ally's and my rooms will take hours to heat up

**Reply by jeppper** [reply]
No but I have also noticed that the Aqaras are not precise at all.
I will have a look at the distance adapters though. They might work for me too.

Do you have xperience with the "external open window" setting? The Ally does not seem to react when setting this to OPEN.

**Reply by lux73** [reply]
for sure - the "external open window" Setting even need's to be set from your Home Automation and works flawless

i'm interpret my Window Contacts to send "window open" command to my Ally's

**Reply by jeppper** [reply]
Is there some sort of indication on the Ally that a window is open?
How fast does it react on window open?

**Reply by lux73** [reply]
nope - no indication on the Ally. I have some visual verification my OpenHAB Sitemap showing on my Tablet

reaction is about 20-30 Seconds after opening the Window before the Valve closes completely

**Comment by jeppper** [comment]
Okay
I have noticed now that only if "radiator covered" is set to false it reacts on "external windows open"

**Reply by lux73** [reply]
that's weird :-o

**Reply by jeppper** [reply]
Yes it is.

**Reply by mipoat** [reply]
Thank you so much, it drove me mad that my Popp TRV is not detecting the open window automatically and ignores the "external window open". Setting "radiator covered" to false did the trick for me too.

It is only a pitty, that "radiator covered" = false causes the TRV to obviously use an average value between the internal and external temp sensors. I don't get this averaging feature. For me the internal sensor is just a fallback solution, if there is no external sensor. It is always bad to measure the temp so close to the radiator. 
If "radiator convered" = true, my TRV uses only the external temp sensor, which is the right way to do it. But, in this mode there is just no open window detection. 
It seems, you just can't have it all.
